src/fluree/db/server_settings.clj

```diff
@@ -445,39 +445,37 @@
-                                         :novelty-max       (-> settings :fdb-memory-reindex-max env-bytes)
-                                         :dev?              dev?
-                                         :password-auth     (password-feature-settings settings)
-                                         :open-api          (-> settings :fdb-api-open env-boolean)
-                                         :file-storage-path (when (= :file storage-type)
-                                                              file-ledger-storage-path)
-                                         :s3-storage        (when (= :s3 storage-type)
-                                                              {:bucket (:bucket s3-conn)
-                                                               :prefix s3-ledger-storage-prefix})
-                                         :encryption-secret encryption-key}))}))
+     :options      {:transactor?    is-transactor?
+                    :tx-private-key (get-or-generate-tx-private-key settings)
+                    :storage-read storage-read
+                    :storage-exists storage-exists
+                    ;; Storage write is overwritten in the default transactor to
+                    ;; use the RAFT group as part of the write process
+                    :storage-write storage-write
+                    :storage-rename storage-rename
+                    :storage-list storage-list
+
+                    ;; create our own request channel so we can monitor it if in 'dev' mode
+                    :req-chan (async/chan)
+
+                    :memory (some-> settings :fdb-memory-cache env-bytes)
+                    :close-fn close-fn
+                    :serializer serializer
+                    ;; ledger-specific settings meta is a map of settings that
+                    ;; are implementation-specific, i.e.  a transactor needs
+                    ;; novelty-min and novelty-max, a web browser connection
+                    ;; might need some different info
+                    :meta {:novelty-min       (-> settings :fdb-memory-reindex env-bytes)
+                           :novelty-max       (-> settings :fdb-memory-reindex-max env-bytes)
+                           :dev?              dev?
+                           :password-auth     (password-feature-settings settings)
+                           :open-api          (-> settings :fdb-api-open env-boolean)
+                           :file-storage-path (when (= :file storage-type)
+                                                file-ledger-storage-path)
+                           :s3-storage        (when (= :s3 storage-type)
+                                                {:bucket (:bucket s3-conn)
+                                                 :prefix s3-ledger-storage-prefix})
+                           :encryption-secret encryption-key}}}))
```

Comment:
Needs some alignment here too

Reply:
I feel a little silly but I don't see what's unaligned here. Happy to change it, but I'm not sure what to change : )

Reply:
Sorry, it's up above here. I just scrolled to the bottom of the whole map to make my comment.

Reply:
Oh wait, I see that you must mean the whole `generate-conn-settings` function body.

Reply:
...and in doing so scrolled to the next file hahaha. It's in dev/user actually. My bad!

Reply:
Oh wait, GitHub was hiding things from me again. Yes, I mean this file but lines 450-468. Sorry I'm making this extra confusing now!